source/configuration_manual/dovecot_ssl_configuration.rst

```diff
@@ -125,7 +125,7 @@ If you have multiple IPs available, this method is guaranteed to work with all c
 With client TLS SNI (Server Name Indication) support
 ****************************************************
 
-It is important to note that having multiple SSL certificates per IP will not be compatible with all clients, especially mobile ones. It is a TLS SNI limitation. See SSL/SNIClientSupport for list of clients known to (not) support SNI.
+It is important to note that having multiple SSL certificates per IP will not be compatible with all clients, especially mobile ones. It is a TLS SNI limitation. See https://wiki.dovecot.org/SSL/SNIClientSupport for list of clients known to (not) support SNI.
```

Comment:
it would be better to move the page too.

Reply:
wasn't sure if that was desired - I'll transfer the contents. thx!

Reply:
hi, did you have a chance to look at this?

Reply:
thanks for your patience - copied contents now.